bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -21,23 +36,215 @@ log:set DEBUG org.openhab.core.automation
 
-```js
-const LoggerFactory = Java.type('org.slf4j.LoggerFactory');
-
-LoggerFactory.getLogger("org.openhab.core.automation.examples").info("Hello world!");
+```javascript
+let logger = Java.type('org.slf4j.LoggerFactory').getLogger('org.openhab.rule.' + this.ruleUID);
+logger.info('Hello world!');
+logger.warn('Successfully logged warning.');
+logger.error('Successfully logged error.');
 ```
 
-Depending on the openHAB logging configuration, you may need to prefix logger names with `org.openhab.core.automation` for them to show up in the log file (or you modify the logging configuration).
-
 The script uses the [LoggerFactory](https://www.slf4j.org/apidocs/org/slf4j/Logger.html) to obtain a named logger and then logs a message like:
 
 ```text
-    ... [INFO ] [.openhab.core.automation.examples:-2   ] - Hello world!
+    ... [INFO ] [org.openhab.rule.<ruleId>   ] - Hello world!
+    ... [WARN ] [org.openhab.rule.<ruleId>   ] - Successfully logged warning.
+    ... [ERROR] [org.openhab.rule.<ruleId>   ] - Successfully logged error.
+```
+
+## Core Actions
+
+The openHAB services, which are pre-included in the integrated JavaScript engine, must explicitely be imported.
+
+__Please note:__ The [helper library](https://github.com/openhab/openhab-js) is on the way and will become the preferred API to work with openHAB.
+
+```javascript
+let openhab = require('@runtime');
+```
+
+### itemRegistry
+
+```javascript
+let state = openhab.itemRegistry.getItem(itemName).getState();
+```
+
+You can use `toString()` to convert an item's state to string or `toBigDecimal()` to convert to number.
+
+### Event Bus Actions
+
+```javascript
+openhab.events.sendCommand(itemName, command);
+openhab.events.postUpdate(itemName, state);
+```
+
+`command` and `state` can be a string `'string'` or a number depending on the item.
+
+### Exec Actions
+
+Execute a command line.
+
+```javascript
+openhab.Exec = Java.type('org.openhab.core.model.script.actions.Exec');
+let Duration = Java.type('java.time.Duration');
+
+// Execute command line.
+openhab.Exec.executeCommandLine('echo', 'Hello Wordl!');
+
+// Execute command line with timeout.
+openhab.Exec.executeCommandLine(Duration.ofSeconds(20), 'echo', 'Hello Wordl!');
+
+// Get response from command line.
+let response = openhab.Exec.executeCommandLine('echo', 'Hello Wordl!');
+
+// Get response from command line with timeout.
+response = openhab.Exec.executeCommandLine(Duration.ofSeconds(20), 'echo', 'Hello Wordl!');
+```
+
+### HTTP Actions
+
+For available actions have a look at the [HTTP Actions Docs](https://www.openhab.org/docs/configuration/actions.html#http-actions).
+
+```javascript
+openhab.HTTP = Java.type('org.openhab.core.model.script.actions.HTTP');
+
+// Example GET Request
+var response = openhab.HTTP.sendHttpGetRequest('<url>');
+```
+
+Replace `<url>` with the request url.
+
+### Timers
+
+```javascript
+let ZonedDateTime = Java.type('java.time.ZonedDateTime');
```

Comment:
FYI in addition to the console logging, this binding is very shortly going to support Javascript timers,  again see https://github.com/digitaldan/openhab-addons/blob/f99e8affce197a95d4b829b31b1907cefce10819/bundles/org.openhab.automation.jsscripting/src/main/resources/node_modules/@globals.js for how we are doing this.